janusgraph-test/src/main/java/org/janusgraph/blueprints/AbstractJanusGraphProvider.java

```diff
@@ -46,6 +46,9 @@
 import org.janusgraph.graphdb.vertices.CacheVertex;
 import org.janusgraph.graphdb.vertices.PreloadedVertex;
 import org.janusgraph.graphdb.vertices.StandardVertex;
+
+import com.google.common.collect.Sets;
```

Comment:
Did your ide insert this here? I'd expect this to go before or after but not in between the org imports.